lib/deploy-spa.js

```diff
@@ -11,13 +11,20 @@ async function deploy(settings) {
     settings.hashFileNames
   );
 
+  const scripts = assetsWithoutContent.filter(x => x.type === 'script');
+  const styleSheets = assetsWithoutContent.filter(x => x.type === 'styleSheet');
+
   const spa = {
     name: settings.name,
     package_config: settings.packageConfig,
-    scripts: assetsWithoutContent,
+    scripts,
     sky_ux_config: settings.skyuxConfig
   };
 
+  if (styleSheets.length) {
```

Comment:
I left a similar comment on the Angular builders PR about casing this as `stylesheet`.